.test_runner_config.yaml

```diff
@@ -53,6 +53,11 @@ steps:
   lint:
   - make PYTHON=/usr/bin/python2 V=0 lint
   - make PYTHON=/usr/bin/python3 V=0 pylint
+  webui_unit:
+  - dnf install -y npm
+  - cd ${container_working_dir}/install/ui/js/libs && make
```

Comment:
``make -C ${container_working_dir}/install/ui/js/libs``

Reply:
Thank you for the tip, I did not know about this option. But for now I keep it my way, because if we would like to change it we also need to change ipa-docker-test-runner and do another release and then bump that version in .travis.yml file here in repository. And I think that is not a big deal to keep it this way.